_data/navbars/getting-started.yml

```diff
@@ -119,6 +119,11 @@ section:
       path: /getting-started/kubernetes/hardway/istio-integration
   - title: System requirements
     path: /getting-started/kubernetes/requirements
+  - title: Operator customizations
+    path: /getting-started/kubernetes/customizations
+    section:
+      - title: Deploying Images by Digest
```

Comment:
nit: should use sentence case for titles (so only capitalize the first word and proper nouns)

Reply:
Deploy images by digest